tests/integration/docusaurus/connecting_to_your_data/database/util.py

```diff
@@ -1,16 +1,25 @@
+from sqlalchemy.exc import SQLAlchemyError
+
+
 def load_data_into_database(
     table_name: str, csv_path: str, connection_string: str
 ) -> None:
     import pandas as pd
     import sqlalchemy as sa
 
     engine = sa.create_engine(connection_string)
-    engine.execute(f"DROP TABLE IF EXISTS {table_name}")
-    print(f"Dropping table {table_name}")
-    df = pd.read_csv(csv_path)
-    df = df.head(
-        10
-    )  # <WILL> This line is here to address performance issues we have been running into with cloud resources (ie redshift). Can be taken out
-    print(f"Creating table {table_name} from {csv_path}")
-    df.to_sql(name=table_name, con=engine, index=False)
-    engine.dispose()
+    try:
+        connection = engine.connect()
+        print(f"Dropping table {table_name}")
+        connection.execute(f"DROP TABLE IF EXISTS {table_name}")
+        df = pd.read_csv(csv_path)
+        # helping performance
+        df = df.head(10)
+        print(f"Creating table {table_name} from {csv_path}")
+        df.to_sql(name=table_name, con=engine, index=False)
+    except SQLAlchemyError as e:
+        # TODO Add logging messages here
```

Comment:
Although we don't have our `logger` object instantiated just yet, can we put a `logging.error` or `logging.warning` here for the time being?